mod/01_core/set/all/content.rb

```diff
@@ -19,8 +19,11 @@ def raw_content
   structure ? template.db_content : db_content
 end
 
-def chunk_list #override to customize by set
-  :default
+
+format :all do
+  def chunk_list #override to customize by set
+    :default
+  end
```

Comment:
Thinking about this further, I sort of see the need for this, but not in this case.  The set/all modules get included into the format class and will be inherited.  But for other sets where they get loaded onto the singleton class (i.e. format instances for format declarations), I'm not sure.  I'd have to look at the code some more.  It would be nice to have a test that would fail if it does require this kind of declaration.  That would be something like testing if a method or view declared on a non-all set in the base format class is available in subclasses.

In other words, I'm not completely against this method of including a format block into multiple formatters, but we should consider 1) is it needed, and 2) is there a better way to do this in the singleton module inclusion process.